ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivGetTransactionCountTest.java

```diff
@@ -35,18 +42,30 @@
   private final String privacyGroupId =
       BytesValues.asBase64String(BytesValue.wrap("0x123".getBytes(UTF_8)));
 
+  private final String privateFrom = "thePrivateFromKey";
+  private final String[] privateFor = new String[] {"first", "second", "third"};
+  private final Address address = Address.fromHexString("55");
+
+  private JsonRpcRequest legacyRequest;
+
   private final Address senderAddress =
       Address.fromHexString("0x627306090abab3a6e1400e9345bc60c78a8bef57");
   private final long NONCE = 5;
 
+  @Before
+  public void setup() {
+    final Object[] jsonBody = new Object[] {address.toString(), privateFrom, privateFor};
+    legacyRequest = new JsonRpcRequest("2.0", "priv_getTransactionCount", jsonBody);
+  }
+
   @Test
   public void verifyTransactionCount() {
```

Comment:
Thinking that this test should be renamed now to indicate it is for privacyGroup transaction count. And can another test be added for eea groups as well?

Reply:
Changed the names of the tests.